documents/operating.md

```diff
@@ -41,13 +41,90 @@ SECTION 3: 🤔🤔🤔 Projects 🤔🤔🤔
 ---------------------------------------------------------------------
 SECTION 4: 💵💵💵 MONEY 💵💵💵
 ---------------------------------------------------------------------
-- [ ] Capital contributions
-- [ ] Allocating resources to projects
-- [ ] Distributing profits
-- [ ] Compensation
-- [ ] Expenses
-- [ ] Taxes
-- [ ] Company canoe
+- [x] Capital contributions
+- [x] Allocating resources to projects
+- [x] Distributing profits
+- [x] Compensation
+- [x] Expenses
+- [x] Taxes
+- [x] Company canoe
+
+
+### Capital Contributions
+Each Corporate Overlord will contribute an $300 initial capital contribution
+to BIFFUD within three months of achieving Corporate Overlord status.
```

Comment:
Where does the $300 number come from? I know you (Dan) wanted ~$3000 to start, based on ~ten COs, so I assume that was the rationale behind it. But we're at almost 20 COs already, likely with more to trickle in before we launch. Should we stick to the $3000 and have all COs pay in less?

If so, should this number reflect "ground level" CO initial contribution as well as that required of future COs? Or is this meant to be static for all future COs rather than the "ground level" ones?

If not, we're way over even the extended theoretical operating budget. Not that that's an inherently bad idea (uh-oh), but I felt it was worth voicing.

(The point of this is that I'm astonished by how many people are signing on to BIFFUD, which is awesome.)